dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```
@@ -168,9 +171,20 @@ public Response toolsIndexGet(SecurityContext securityContext) throws NotFoundEx
                 });
 
                 //index a document
-                HttpEntity entity = new NStringEntity(builder.toString(), ContentType.APPLICATION_JSON);
+                try {
+                    restClient.performRequest("DELETE", "/entry");
+                } catch (Exception e) {
+                    e.printStackTrace();
```

Sure we want to catch all exceptions?